pkg/api/index.go

```diff
@@ -320,9 +320,8 @@ func (hs *HTTPServer) getNavTree(c *models.ReqContext, hasEditPerm bool, prefs *
 			Text:        "Service accounts",
 			Id:          "serviceaccounts",
 			Description: "Manage service accounts",
-			// TODO: change icon to "key-skeleton-alt" when it's available
-			Icon: "keyhole-circle",
-			Url:  hs.Cfg.AppSubURL + "/org/serviceaccounts",
+			Icon:        "key-skeleton-alt",
```

Comment:
```suggestion
			Icon:        "keyhole-circle",
```

this comment was misleading, when we have a new icon 

we should just remove the comment and change this back

<img width="398" alt="image" src="https://user-images.githubusercontent.com/7727602/170903852-ef464617-2c77-4e60-8ae4-7139afdb3d95.png">

Reply:
this is a picture of the new icons we are working with. Until then we should use `keyhole-circle`